src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -223,6 +240,15 @@ private void SetIds(DatabaseTopology topology, string shardedDatabaseId)
 
         public string Name { get; }
 
+        private class ShardProperties
+        {
+            public int Number;
+            public string DatabaseName;
+        }
+
+        private ShardProperties _shardProperties;
+        private bool IsShard => _shardProperties != null;
+        
```

Comment:
@ppekrol those keep repeat it self...
I wonder if we should go the path of having `ShardedDocumentDatabase : DocumentDatabase`

Reply:
k